SIGNING.md

```diff
@@ -0,0 +1,95 @@
+# Signature support
+
+Binstall supports verifying signatures of downloaded files.
+At the moment, only one algorithm is supported, but this is expected to improve as time goes.
+
+This feature requires adding to the Cargo.toml metadata: no autodiscovery here!
+
+## Minimal example
+
+Generate a [minisign](https://jedisct1.github.io/minisign/) keypair:
+
+```console
+minisign -G -p signing.pub -s signing.key
```

Comment:
`minisign` by default uses a password to encrypt the private key.

Simply giving an empty password would still needs stdin to be a terminal to provide a password.

I think we need to mention `-W` as an option here:

```
minisign -W -G -p signing.pub -s signing.key
```